app/Locale/de_DE/LC_MESSAGES/default.po

```diff
@@ -15,292 +15,292 @@ msgstr ""
 "Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
 msgid "Cannot load configuration files with ../ in them."
-msgstr ""
+msgstr "Kann keine Konfigurationsdaten mit ../ darin laden."
 
 msgid "No variable %s found in %s"
-msgstr ""
+msgstr "Keine Variable %s in %s gefunden"
 
 msgid "Encountered unexpected %s. Cannot generate SQL log."
-msgstr ""
+msgstr "Auf unerwartetes %s gestoßen."
 
 msgid "CakePHP: the rapid development php framework"
-msgstr ""
+msgstr "CakePHP: das PHP framework zur schnellen Entwicklung"
 
 msgid "The request has been black-holed"
-msgstr ""
+msgstr "Anfrage ge-black-holed"
 
 msgid "Debug setting does not allow access to this url."
-msgstr ""
+msgstr "Debug-Einstellung erlaubt keinen Zugriff auf diese url."
 
 msgid "Cache engine %s is not available."
-msgstr ""
+msgstr "Cache engine %s nicht verfügbar."
 
 msgid "Cache engines must use %s as a base class."
-msgstr ""
+msgstr "Cache engines müssen %s als Basisklasse verwenden."
 
 msgid "Cache engine \"%s\" is not properly configured. Ensure required extensions are installed, and credentials/permissions are correct"
-msgstr ""
+msgstr "Cache engine \"%s\" ist nicht korrekt konfiguriert. Stellen Sie sicher, dass die nötigen Erweiterungen installiert sind, und dass die Zugriffsrechte richtig sind"
 
 msgid "%s cache was unable to write '%s' to %s cache"
-msgstr ""
+msgstr "%s cache konnte '%s' nicht ins %s cache schreiben"
 
 msgid "Invalid cache group %s"
-msgstr ""
+msgstr "Ungültige cache-Gruppe %s"
 
 msgid "Files cannot be atomically decremented."
-msgstr ""
+msgstr "Dateien können nicht atomisch vermindert werden."
```

Comment:
automatisch

Reply:
Wouldn't the English word for 'automatisch' be 'automatically'? It says 'atomically', which I would translate as 'atomar' or 'atomisch'. I believe this refers to the atomical decrement, e.g. `$var--`